app/models/issue_template.rb

```diff
@@ -16,8 +16,21 @@ class IssueTemplate < ActiveRecord::Base
                   'enabled_sharing','visible_children', 'position'
   attr_accessible :title, :tracker_id, :issue_title, :description, :note,
                   :is_default, :enabled, :enabled_sharing, :author, :project, :position
+
+  scope :enabled_sharing, lambda { where(enabled_sharing: true) }
+  scope :enabled, lambda { where(enabled: true) }
+  scope :is_default, lambda { where(is_default: true) }
+  scope :not_default, lambda { where(is_default: false) }

```

Comment:
__[rubocop]__
* Use the `-> { ... }` lambda literal syntax for single line lambdas. [:ref](https://github.com/bbatsov/ruby-style-guide#lambda-multi-line)